.github/workflows/tests.yml

```diff
@@ -28,7 +28,7 @@ permissions:
 jobs:
   build:
     if: github.event_name == 'pull_request' || github.repository != 'discourse/discourse-private-mirror'
-    name: ${{ matrix.target }} ${{ matrix.build_type }} # Update fetch-job-id step if changing this
+    name: ${{ matrix.target }} ${{ matrix.build_type }}${{ (matrix.target == 'core' && matrix.build_type == 'frontend' && format(' ({0})', matrix.browser)) || '' }} # Update fetch-job-id step if changing this
```

Comment:
per comment - should you add `matrix.browser` to fetch-job-id?

Reply:
Technically it's not needed, since my name change doesn't affect any RSpec jobs. But still, consistency is good, so I've updated it :+1: